src/components/templates/PartyMap/components/RoomCard/RoomCard.tsx

```diff
@@ -1,9 +1,10 @@
 import React from "react";
-import "./RoomCard.scss";
-import RoomAttendance from "../RoomAttendance";
-import { formatMinute } from "utils/time";
+
+import { formatMinute, getCurrentEvent } from "utils/time";
 import { RoomData } from "types/RoomData";
-import { getCurrentEvent } from "utils/time";
+
+import { RoomAttendance } from "..";
```

Comment:
Are you sure this is correct? I've never seen this syntax ".." before and I'm just asking.

Reply:
It's correct, but it may not be the best/easiest to read style. It's essentially accessing `../index.ts` (aka `src/components/templates/PartyMap/components/index.ts`)

Something we can add to our code style discussions in future, and decide on our official standpoint (or let an opinionated linter do it for us)